src/client/components/Navigation/Topnav.js

```diff
@@ -98,9 +135,14 @@ class Topnav extends React.Component {
   };
 
   menuForLoggedIn = () => {
-    const { intl, username } = this.props;
-    const { searchBarActive } = this.state;
-    const { popoverVisible } = this.state;
+    const { intl, username, notifications, userSCMetaData, loadingNotifications } = this.props;
+    const { searchBarActive, notificationsPopoverVisible, popoverVisible } = this.state;
+    const lastSeenTimestamp = _.get(userSCMetaData, 'notifications_last_timestamp');
+    const notificationsCount = _.isUndefined(lastSeenTimestamp)
+      ? _.size(notifications)
+      : _.filter(notifications, notification => lastSeenTimestamp < notification.timestamp).length;
```

Comment:
We use two different things to get basically the same thing (`_.size`) and `.length`). I think we should use one of those methods for both cases.

Reply:
yeah you're right, I prefer to`_.size` since it has a bunch of polyfills + error checks that accompany it. will update now